admin/src/main/java/io/strimzi/arguments/topic/FetchOffsetsCommand.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.arguments.topic;
+
+import io.strimzi.admin.AdminProperties;
+import io.strimzi.arguments.BasicCommand;
+import io.strimzi.utils.FetchOffsetsUtils;
+import io.strimzi.utils.OutputFormat;
+import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.clients.admin.DescribeTopicsResult;
+import org.apache.kafka.clients.admin.ListOffsetsResult;
+import org.apache.kafka.clients.admin.OffsetSpec;
+import org.apache.kafka.clients.admin.TopicDescription;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.TopicPartitionInfo;
+import org.apache.kafka.common.requests.ListOffsetsRequest;
+import picocli.CommandLine;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Command for fetch offsets for topic in Kafka.
+ * Accessed using `admin-client topic fetch-offsets`
+ */
+@CommandLine.Command(name = "fetch-offsets")
+public class FetchOffsetsCommand extends BasicCommand {
+
+    @CommandLine.Option(names = {"--topic", "-t"}, description = "Name of the topic to be inspected.", required = true)
+    String topicName;
+
+    @CommandLine.Option(names = {"--timestamp", "--time"}, description = "Timestamp marking the cutoff point before which offsets will be fetched. See kafka-get-offsets tool for more info.")
+    String timestamp = "latest";
```

Comment:
The approach how default variable is a bit different in outputFormat and timestamp. would it be possible to unify it ?  or is there something to gain from these different approaches ? 

e.g.
```
@CommandLine.Option(names = {"--timestamp", "--time"}, defaultValue = "latest", description = "Timestamp marking the cutoff point before which offsets will be fetched. See kafka-get-offsets tool for more info.")
String timestamp;
```

Reply:
Good point, I simply just missed the possibility to define it directly in Option, code updated now.